### PR TITLE
first commit for block and like page

### PR DESCRIPTION
**Pull request by sunbaijing**

### CODE REVIEW

File: src/component/BlockList.js [hunk 1]
```diff
@@ -1,23 +1,61 @@
...
-                    </Col>
-                )}
-            </Row>
-        </div>
-    )
-}
+  return (
+    <div>
+      <h1>Block List</h1>
+      <Row xs={1} md={4} lg={5} className="block-list">
```
Comment (TT0-GIT) [inline src/component/BlockList.js:23]: I recommend use Container to wrap Row, which will give margin for left and right

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import Navigation from './component/Navigation.js';
-import LikeList from './component/LikeList.js';
-import MovieList from './component/MovieList.js';
-import BlockList from './component/BlockList.js'
-import Home from './component/Home.js'
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (TT0-GIT) [inline src/App.js:3]: you don't have to import "bootstrap/dist/css/bootstrap.min.css" in every script, I already import in index.js

File: package.json [hunk 3]
```diff
@@ -1,38 +1,33 @@
...
-    "react-bootstrap": "^1.0.1",
-    "react-dom": "^16.13.1",
-    "react-route": "^1.0.3",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "bootstrap": "4.5.0",
+    "react": "16.12.0",
+    "react-bootstrap": "1.0.1",
+    "react-dom": "16.12.0",
+    "react-router-dom": "5.2.0",
```
Comment (TT0-GIT) [inline package.json:15]: recommend keep Semantic Versioning (with ^). You can check on this [Semver explained - why is there a caret (^) in my package.json?](https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/)

File: public/public/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <title>Top Rated Movie List</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+  </body>
+</html>
```
Comment (TT0-GIT) [inline public/public/index.html:17]: you don't have to put another public folder in public.

**Comment by sunbaijing** [comment]
Just upload again

**Pull request merged**